shared/src/lib.rs

```diff
@@ -18,10 +18,6 @@ pub mod wg;
 pub use types::*;
 
 lazy_static! {
-    pub static ref CLIENT_CONFIG_DIR: &'static Path = Path::new("/etc/innernet");
-    pub static ref CLIENT_DATA_DIR: &'static Path = Path::new("/var/lib/innernet");
-    pub static ref SERVER_CONFIG_DIR: &'static Path = Path::new("/etc/innernet-server");
-    pub static ref SERVER_DATABASE_DIR: &'static Path = Path::new("/var/lib/innernet-server");
     pub static ref REDEEM_TRANSITION_WAIT: Duration = Duration::from_secs(5);
 }
```

Comment:
Optional, because this was not introduced by this PR: `REDEEM_TRANSITION_WAIT` may be made just a `const REDEEM_TRANSITION_WAIT: Duration = Duration::from_secs(5);` without the `lazy_static`?
https://doc.rust-lang.org/std/time/struct.Duration.html#method.from_secs is const from rustc 1.32.

Reply:
nice catch! will fix :)